docs/2.0/dynamo/troubleshooting.html

```diff
@@ -1083,7 +1083,7 @@ <h2>File an Issue<a class="headerlink" href="#file-an-issue" title="Permalink to
 <p>If you experience problems with TorchDynamo, <a class="reference external" href="https://github.com/pytorch/torchdynamo/issues">file a GitHub
 issue</a>.</p>
 <p>Before filing an issue, read over the <a class="reference external" href="../README.md">README</a>,
-<a class="reference external" href="./TROUBLESHOOTING.md">TROUBLESHOOTING</a>, and search for similar
+<a class="reference external" href="../_sources/dynamo/troubleshooting.rst.txt">TROUBLESHOOTING</a>, and search for similar
 issues.</p>
```

Comment:
We should just remove these lines as this information is no longer present.

Reply:
Thank you for your feedback, @svekars. I wanted to clarify the suggested change. Are you recommending the removal of the entire line 'Before filing an issue, read over the README, TROUBLESHOOTING, and search for similar issues'? I just want to ensure I understand your suggestion correctly before making the modification. Please let me know. Thank you!